packages/@aws-cdk-testing/cli-integ/lib/aws.ts

```diff
@@ -29,10 +29,6 @@ interface ClientConfig {
 }
 
 export class AwsClients {
-  public static async default(output: NodeJS.WritableStream) {
```

Comment:
No one is using this now - which is great, we really shouldn't have been calling this function at all because it bypasses the region pool.